### PR TITLE
use length of saved_logit array when evaluating logits

### PR DESCRIPTION
**Pull request by nickdnickd**
I ran into this on the first run similar to issue https://github.com/jannisborn/covid19_ultrasound/issues/103
When I only had one fold by default. This should hopefully allow testing of any number of folds.

### CODE REVIEW

**Review by jannisborn — APPROVED** [review]

**Comment by jannisborn** [comment]
Thanks again, makes perfect sense!
